scenarios/mongoose_muc_light.erl

```diff
@@ -0,0 +1,304 @@
+            timer:sleep(100),
+            get_coordinator_pid()
+    end.
+
+%%------------------------------------------------------------------------------------------------
+%% User
+%%------------------------------------------------------------------------------------------------
+start_user(Client) ->
+    lager:debug("User process ~p", [self()]),
+    erlang:monitor(process, Client#client.rcv_pid),
+    create_muc_light_room(Client),
+    escalus_tcp:set_active(Client#client.rcv_pid, true),
+    send_jid_and_pid_to_coordinator(Client),
+    user_loop(Client, 0).
+
+send_jid_and_pid_to_coordinator(Client) ->
+    Coordinator = get_coordinator_pid(),
+    lager:debug("Process ~p sending pid and jid to coordinator ~p", [self(), Coordinator]),
+    Jid = escalus_client:short_jid(Client),
+    Coordinator ! {new_client, self(), Jid}.
+
+user_loop(Client, Time) ->
+    receive
+        publish_item ->
+            amoc_metrics:update_counter(message_published, 1),
+            Now = os:system_time(microsecond),
+            send_message_to_room(Client),
+            update_publication_interval(Time, Now),
+            user_loop(Client, Now);
+        {publish_item, RoomJid} ->
+            amoc_metrics:update_counter(message_published, 1),
+            Now = os:system_time(microsecond),
+            send_message_to_room(Client, RoomJid),
+            update_publication_interval(Time, Now),
+            user_loop(Client, Now);
+        {stanza, _, #xmlel{name = <<"message">>} = Stanza, #{recv_timestamp := TimeStamp}} ->
+            Now = os:system_time(microsecond),
+            BareJid = get_sender_bare_jid(Stanza),
+            schedule_publishing(self(), BareJid),
+            amoc_metrics:update_counter(message_received, 1),
+            update_message_ttd(TimeStamp, Now),
+            user_loop(Client, Time);
+        {add_users, MemberJids} ->
+            add_users_to_room(Client, MemberJids),
+            user_loop(Client, Time)
+    end.
+
+schedule_publishing(Pid, RoomJid) ->
+    amoc_throttle:send(?PUBLICATION_THROTTLING, Pid, {publish_item, RoomJid}).
+schedule_publishing(Pid) ->
+    amoc_throttle:send(?PUBLICATION_THROTTLING, Pid, publish_item).
+
+update_publication_interval(0, _Now) ->
+    ok;
+update_publication_interval(Time, Now) ->
+    Interval = Now - Time,
+    amoc_metrics:update_time(publication_interval, Interval).
+
+update_message_ttd(0, _Now) ->
+    ok;
+update_message_ttd(Time, Now) ->
+    Interval = Now - Time,
+    amoc_metrics:update_time(message_ttd, Interval).
+
+%%------------------------------------------------------------------------------------------------
+%% User connection
+%%------------------------------------------------------------------------------------------------
+connect_amoc_user(Id) ->
+    Cfg = make_user_cfg(Id),
+    {ok, Client, _} = escalus_connection:start(Cfg),
+    send_presence_available(Client),
+    erlang:put(jid, Client#client.jid),
+    Client.
+
+make_user_cfg(Id) ->
```

Comment:
maybe move it to `config` module?

Reply:
Makes sense, although maybe different tests require different configs? I just followed some other test here, but moving it seems reasonable. I'll see what Denys has to say about that

Reply:
Than they will add parameters to the function.
I am not saying everything has to be perfect instantaneously, AFAIR some later on refactoring will be done, so I guess it can be done than :)

Reply:
please don't. config module is the general helper module. it has just one and the only purpose - to extract and verify environment configuration required for the scenario.